### PR TITLE
Update detectors.md

### PR DESCRIPTION
**Pull request by digiblur**
Add example for dual PCIe

### CODE REVIEW

File: docs/docs/configuration/detectors.md [hunk 1]
```diff
@@ -30,6 +30,18 @@ detectors:
     device: usb:1
 ```
 
+Multiple PCIE/M.2 Corals:
```
Comment (debsahu) [inline docs/docs/configuration/detectors.md:33]: Please add comment that this only works when "Protection Mode" is off.

Comment (digiblur) [reply docs/docs/configuration/detectors.md:33]: Protection mode? Not familiar with that.

Comment (debsahu) [reply docs/docs/configuration/detectors.md:33]: Supervisor addon has that option
image

Comment (debsahu) [reply docs/docs/configuration/detectors.md:33]: This maybe specific to hassio addon as my guess is only /dev/apex_0 is passed under protection mode: https://github.com/blakeblackshear/frigate-hass-addons/blob/4256ca2731277dd9b84b02b42b91053414f144e4/frigate/config.json#L26

Comment (digiblur) [reply docs/docs/configuration/detectors.md:33]: I don't have supervisor to test it or comment about the use of said feature.

Comment (digiblur) [reply docs/docs/configuration/detectors.md:33]: Looks like a feature needs to be added to that to allow the user to modify the mappings.

**Pull request merged**